src/meltano/cli/cli.py

```diff
@@ -64,7 +63,7 @@ def main(self, *args, **kwargs) -> NoReturn:
     type=click.Path(exists=True, file_okay=False, resolve_path=True, path_type=Path),
     help="Run Meltano as if it had been started in the specified directory.",
 )
-@click.version_option(version=meltano.__version__, prog_name="meltano")
+@click.version_option(prog_name="meltano")
```

Comment:
Would this help `click` figure out the version slightly faster?

```suggestion
@click.version_option(prog_name="meltano", package_name="meltano")
```

https://github.com/pallets/click/blob/cba52fa76135af2edf46c154203b47106f898eb3/src/click/decorators.py#L460-L475